.github/workflows/ci.yaml

```diff
@@ -15,13 +15,12 @@ jobs:
         with:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
-      - uses: actions/setup-java@v3
-        with:
```

Comment:
the new action will look at `.tool-versions` to get the java version, but this file already exists and matches the value specified here (corretto 11), although it's pinned to a specific patch version.